seisidd/experiment.py

```diff
@@ -446,8 +452,8 @@ def tomo_scan(self, cfg):
         det                 = self.tomo_det
         tomostage           = self.tomo_stage
         # TODO: commented for Sim test
-        # shutter             = self.shutter
-        # shutter_suspender   = self.suspend_shutter
+        shutter             = self.shutter
+        shutter_suspender   = self.suspend_shutter
```

Comment:
we might need a better way to handle sim and real detector cases in the future.  For the upcoming run, this should be sufficient.